lib/modules/datasource/artifactory/__fixtures__/releases-as-files.html

```diff
@@ -10,10 +10,10 @@ <h1>Index</h1>
        <pre>Name      Last modified      Size</pre><hr/>
        <pre>
          <a href="..">..</a>
-         <a href="1.0.0">1.0.0</a>  21-Jul-2021 20:08    -
-         <a href="1.0.1">1.0.1</a>  23-Aug-2021 20:03    -
-         <a href="1.0.2">1.0.2</a>  21-Jul-2021 20:09    -
-         <a href="1.0.3">1.0.3</a>  06-Feb-2021 09:54    -
+         <a href="1.0.0">1.0.0</a>  21-Jul-2021 20:08    2.00 GB
+         <a href="1.0.1">1.0.1</a>  23-Aug-2021 20:03    12 MB
+         <a href="1.0.2">1.0.2</a>  21-Jul-2021 20:09    123.45 GB
+         <a href="1.0.3">1.0.3</a>  06-Feb-2021 09:54    9.0 KB
```

Comment:
Are they always like this? i.e. the previous test fixture was _wrong_?
Or should we retain some lines which are missing this last part?

Reply:
I'm unfortunately not familiar enough with Artifactory to know if it was *always* like this, just that it's like this on our instance (Artifactory/7.84.15). The previous test fixture matches the format for directories but not for files (I guess the recursive size computation for directories isn't worth it?).

Keeping some of the lines in the old format could be a good idea, in case this is representative of other Artifactory instances.

Reply:
Great, please do that